deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql

```diff
@@ -1,4 +1,4 @@
-CALL apoc.load.json('file:/tmp/mongo-export/splits/current-chunk.json') YIELD value as badge
+CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL}") YIELD value as badge
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-28T17:33:24Z" title="Tuesday, May 28th 2019, 7:33:24 pm +02:00">May 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,4 +1,4 @@
-CALL apoc.load.json('file:/tmp/mongo-export/splits/current-chunk.json') YIELD value as badge
+CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL}") YIELD value as badge
```


I don't think you can expand variables like this. It's a `.cql` file, not a bash script.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-28T19:05:40Z" title="Tuesday, May 28th 2019, 9:05:40 pm +02:00">May 28, 2019</time>_

---


You can if you use magic like I do here: https://github.com/Human-Connection/Human-Connection/pull/697/files#diff-c6d49070ce711f563a75e723b1e8682cR14